_posts/2016-10-10-visualise-interruptions-with-a-lego-wall.md

```diff
@@ -0,0 +1,119 @@
+
+The category “Help the PO” encompasses the activities that the development team is asked to do to help the PO prioritise stories (backlog refinement sessions). In fact these activities should be considered part of the work, but too often the development team perceives this activities as interruptions. Work is not only about coding, ux design or testing; it should encompass collaboration with the PO because these activities are essential to delivering value.
+
+
+##Materials and preparation
+
+{% img /assets/img/custom/blog/2016-10-10-visualise-interruptions-with-a-LEGO-wall/lego-wall-materials.jpg "" %}
+
+You can build a LEGO Wall with as many **base plates** as you need, in order to:
+
+ - Display the days of the Sprint (2 boards are enough to keep track of a 2-week Sprint)
+ - Display as many rows as team members (4 developers as shown in the photos)
+ - A legend of the colours used (4 colours + white)
+
+You will need **coloured LEGO bricks** to display interruptions and **LEGO figures** to represent an anonymous team member.
+
+From the pictures, you can see that the team decided to use LEGO figures to represent themselves. This is because this activity should be pleasurable as well as useful. In fact, I recommend to create your own figure as if you are playing a tabletop game. Thanks to the modularity of LEGO and the figures from movies and comics, you can be whoever you want and change your character as many times as you like, so that you can keep your anonymity.
+
+Lego is an excellent tool to visualise interruptions because:
+
+- Thanks to its modularity, it is easy to split a working day in 16 1x1 blocks. A 1x1 block corresponds to 30 minutes of activity	
+- Maintaining the LEGO Wall is an effortless activity
+- It is easy to get the whole team involved because LEGO bricks are beautiful and fun to play with
+
+
+##How to use it
+
+{% img /assets/img/custom/blog/2016-10-10-visualise-interruptions-with-a-LEGO-wall/lego-wall.jpg"" %}
+
+I suggest you hang the LEGO Wall in a place that would be highly visible to the team and stakeholders. We want to trigger discussions in front of it. The LEGO Wall displays real and meaningful data that can be extremely useful during retrospectives. The more colourful the LEGO Wall gets the more margin of improvements the team and the organisation can work on to deliver more value.
+
+Since the LEGO Wall is not a micro-management tool, at the end of every day, each team member should update it exclusively with bricks assigned to each category of impediment. The white bricks should be added only at the end of the Sprint. They have the function of highlighting the time spend to achieve the Sprint goal so that it’s easy to compare how much time the team has spent at working and how much time the team has spent because of interruptions.

```

Comment:
`easy to compare how much time the team has spent working and how much time has been spent on interruptions.`